cisagov/LME
# Upgrade 1x to 2.0

## 🗣 Description ##
Creates a readme and scripts to easily upgrade from 1x to 2x

### 💭 Motivation and context 
We need a path to upgrade from 1x to 2x. 


## 🧪 Testing 

```
# Either in powershell locally or on azure with the repo checked out
cd LME/testing
./SetupTestbed.ps1 -NumClients 2 -AutoShutdownTime 0000 -AllowedSources "your.ip/32" -l centralus -ResourceGroup LME-yourname-T1 -y
./InstallTestbed.ps1 -ResourceGroup LME-cbaxley-T1  -b release-1.4.0 | Tee-Object -FilePath "./T1.output.log"
```
You will now have a 1.4 release installed.

- ssh to the LS1 server.
- Clone the repo in admin.ackbar's home directory  `cd ~ && git clone https://github.com/cisagov/LME.git`
- Checkout this branch `cd LME && git checkout -t origin/cbaxley-393-upgrade-to-2.0` 
- Read the readme in `scripts/upgrade` and follow its directions
- Login to the server and make sure that the 1x dashboards have installed and have data

## ✅ Pre-approval checklist ##

- [x] Changes are limited to a single goal **AND** 
      the title reflects this in a clear human readable format
- [x] Issue that this PR solves has been selected in the Development section
- [x] I have read and agree to LME's [CONTRIBUTING.md](https://github.com/cisagov/LME/CONTRIBUTING.md) document.
- [x] The PR adheres to LME's requirements in [RELEASES.md](https://github.com/cisagov/LME/RELEASES.md#steps-to-submit-a-PR)

## ✅ Pre-merge Checklist

- [x] PR has been tested and the documentation for testing is above
- [x] Squash and merge all commits into one PR level commit 

## ✅ Post-merge Checklist

- [ ] Delete the branch to keep down number of branches
